nginx/files/security.conf

```diff
@@ -1,2 +1,5 @@
 # Security tweaks for nginx
 server_tokens off;
+
+# Cache SSL Sessions to improve performance and security
+ssl_session_cache shared:SSL:50m;
```

Comment:
Dat is vreemd, werkt voor MainFlow prima: https://github.com/Enrise/Saltmaster/blob/master/salt/environments/prod/files/projects/epic/nginx/sf2_vhost.conf.jinja#L20